enginetest/queries/query_plans.go

```diff
@@ -129,23 +127,23 @@ From xy;`,
 			"         ├─ columns: [xy.x:1!null, xy.y:2]\n" +
 			"         └─ LookupJoin\n" +
 			"             ├─ Distinct\n" +
-			"             │   └─ Project\n" +
-			"             │       ├─ columns: [scalarSubq0.v:1]\n" +
-			"             │       └─ TableAlias(scalarSubq0)\n" +
+			"             │   └─ Distinct\n" +
```

Comment:
Double distinct seems sub-optimal, have a plan to fix?

Reply:
good catch, added a rule now that this is possible